src/modules/completed.js

```diff
@@ -0,0 +1,12 @@
+const completed = (e, tasks) => {
```

Comment:
- [ ] When the `clear all completed` button is pressed, the updated list in the local storage should begin from 1, as for now, it conserves the index from before, kindly consider fixing this so the index is updated upon the removal of a taste using `Clear all completed` button

Reply:
Hey @eaesmat, the issue your previous reviewer asked you to fix still persists. Kindly fix it. As you can see here, the two elements have the same index.
![image](https://user-images.githubusercontent.com/33268927/179176615-84009427-09d3-4d5e-8162-62506a832330.png)
So, when I delete the last element with index 4, the first element with index 4 gets deleted.
I would recommend, resetting the index when an element is deleted. That should solve the issue.